pyrobosim/pyrobosim/navigation/rrt.py

```diff
@@ -25,6 +25,8 @@ def __init__(
         max_nodes_sampled=1000,
         max_time=5.0,
         rewire_radius=1.0,
+        use_grid=False,
+        grid_resolution=0.05,
```

Comment:
Add docstrings if adding these parameter.

Also... how will this work out in the case where we have multiple robots with RRT Planners and they use different resolutions? Should the occupancy grid then be part of the planner rather than the world?

Reply:
Given that this should also deal with multiple robots , that may each use a different resolution for the grid.
It would make more sense to have the occupancy grid as part of the planner.